app/forms/providers/means/regular_outgoings_form.rb

```diff
@@ -112,6 +116,24 @@ def destroy_cash_outgoing_transactions!
           .destroy_all
       end
 
+      def housing_payments_selected?
+        transaction_types.exists?(name: "rent_or_mortgage")
+      end
+
+      def destroy_housing_benefit_transactions!
+        legal_aid_application
+          .legal_aid_application_transaction_types
+          .includes(:transaction_type)
+          .where(transaction_type: { name: "housing_benefit" })
+          .destroy_all
+
+        legal_aid_application
+          .regular_transactions
+          .includes(:transaction_type)
```

Comment:
**nit** I don't believe we need this `include` if we are just trying to destroy the join table record